chatbot/insert/add_quick_replies_to_msg_text_and_localization.js

```diff
@@ -131,271 +168,145 @@ function modify_router_node_cases(flow, node, action, curr_loc, quick_replies, r
 
     let router = routers[dest_id];
     if (router) {
+        // loop through all the cases in the wfr node
```

Comment:
What is 'wfr'?

It is evident from the code that we are looping through all the cases, so I'm not sure that this comment is useful.

Reply:
Reading further, I can see that there are many comments like this that seem to be merely stating what the code is doing. Please review these comments, if you have added them, and think about whether they are actually going to help a future reader of the code to understand it better.

My opinion on comments is that they should be avoided, and only used as a last resort, for example, to explain _why_ the code was written in this particular way (if it might be surprising), or to give some context. If you feel the need to explain what the code does, then that is a strong suggestion that the code may need to be refactored so that it's easier to read and understand.

Reply:
Understood, have gone through and taken out the unnecessary comments